utility/importer/brave_importer.cc

```diff
@@ -39,7 +39,8 @@
 #include "sql/statement.h"
 #include "url/gurl.h"
 
-#include "brave/vendor/bat-native-ledger/src/static_values.h"
+#include \
```

Comment:
this is incorrect, it should just be "bat/ledger/internal/static_values.h"

Reply:
done